Mathlib/Analysis/InnerProductSpace/Positive.lean

```diff
@@ -55,12 +165,20 @@ def IsPositive (T : E →L[𝕜] E) : Prop :=
 theorem IsPositive.isSelfAdjoint {T : E →L[𝕜] E} (hT : IsPositive T) : IsSelfAdjoint T :=
   hT.1
 
-theorem IsPositive.inner_nonneg_left {T : E →L[𝕜] E} (hT : IsPositive T) (x : E) :
+theorem IsPositive.re_inner_nonneg_left {T : E →L[𝕜] E} (hT : IsPositive T) (x : E) :
     0 ≤ re ⟪T x, x⟫ :=
   hT.2 x
 
+theorem IsPositive.re_inner_nonneg_right {T : E →L[𝕜] E} (hT : IsPositive T) (x : E) :
+    0 ≤ re ⟪x, T x⟫ := by rw [inner_re_symm]; exact hT.re_inner_nonneg_left x
+
+@[deprecated re_inner_nonneg_left (since := "2025-05-28")]
```

Comment:
Instead of adding deprecations, add the same lemma `isPositive_iff_isSelfAdjoint_and_inner_nonneg_left` and the others you added above. Then in the PR description, mark these as "MOVES" (details for doing this are located in the information when you edit the PR description)